docs/reference/how-to/disk-usage.asciidoc

```diff
@@ -136,20 +136,42 @@ PUT index
 // CONSOLE
 
 [float]
+=== Watch your shard size
+
+Larger shards are going to be more efficient at storing data. To increase the size of your shards, you can decrease the number of primary shards in an index by <<indices-create-index,creating indices>> with less primary shards, creating less indices (e.g. by leveraging the <<indices-rollover-index,Rollover API>>), or modifying an existing index using the <<indices-shrink-index,Shrink API>>.
+
+Keep in mind that very large shard sizes come with drawbacks, such as longer recovery times.
```

Comment:
Maybe, I'm not sure under which circumstances we'd do a full recovery. Can you suggest a wording?